src/lib/tests/openai/completions.test.ts

```
@@ -0,0 +1,597 @@
+					}
+				}
+			}
+
+			console.log("Streaming tool call - name:", toolCallName);
+			console.log("Streaming tool call - arguments:", toolCallArguments);
+			expect(toolCallName).toBe("calculate");
+			expect(toolCallArguments).toBeDefined();
+
+			// Simulate tool execution
+			const parsedArgs = JSON.parse(toolCallArguments);
+			const calculationResult = eval(parsedArgs.expression); // Simple eval for test
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)